evalml/data_checks/datetime_format_data_check.py

```diff
@@ -36,14 +37,6 @@ def validate(self, X, y):
             >>> datetime_format_dc = DateTimeFormatDataCheck(datetime_column="dates")
             >>> assert datetime_format_dc.validate(X, y) == [
             ...     {
-            ...         "message": "Column 'dates' has datetime values missing between start and end date.",
```

Comment:
Why is this being removed?

Reply:
It's no longer detected by woodwork - it purely errors out unable to detect a frequency, so no more missing values. IMO, this is the more accurate output!